### PR TITLE
Portolio - Mobile Menu

### PR DESCRIPTION
**Pull request by HFG43**
Hi! Thanks for the previous feedback. Have made the following changes as requested:

- [ ] Align Menu and closing button
- [ ] Expand blur, to avoid seeing the rest of the project during the menu displayed
- [ ] Changed the blur container position, to be able to display at full Height and fixed

Thank you very much for reviewing my code!
**--**

Hi there! I'm sending a PR so you can please review my code. For this part of the project, I have included the following:

- [ ] JavaScript Linters added: change spelling to adequate to linter standards
- [ ] Work doing Pair Programing
- [ ] Add Mobile Menu that displays using JS to introduce or to enable CSS classess
- [ ] Followed Figma design
- [ ] Introduced links to menu list items (Contact, About & Portfolio) for better UX using #id's
- [ ] New Mobile buttons are from bigger size, and better contrast for better accessibility
- [ ] Updated README File

Thank you in advance for your feedback!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,18 +11,29 @@
...
+        <span class="ham-bar"></span>
+        </ul>
+      </div>
+      <div class="close-icon"><i class="fa-solid fa-xmark blur"></i></div>
+      <ul class="nav-items-list menu-list desktop-shown">
+        <li><img src="Images/Closing-cross.png" class="cross close-cross"></li>
+        <li><a href="#card-container" class="portfolio">Portfolio</a></li>
+        <li><a href="#about-myself" class="about">About</a></li>
+        <li><a href="#contact-me" class="contact">Contact</a></li>
       </ul>
```
Comment (binhussen) [inline index.html:34]: - Kindly make sure the mobile menu is not scrollable, and your mobile will not be open after the first time, please fix it to have a better user experience, look at the following video to understand what I mean.
https://www.loom.com/share/93ccd3611baa4285ad7bcca65121da4f 👍

File: style.css [hunk 2]
```diff
@@ -741,10 +765,79 @@ textarea::placeholder {
...
+.list-menu-items-font {
+  position: relative;
+  z-index: 30;
+  font-family: 'Poppins', sans-serif;
+  font-size: 32px;
+  font-weight: 600;
+  line-height: 44px;
+  color: #fff;
+  text-decoration: none;
+}
```
Comment (binhussen) [inline style.css:801]: - Kindly, add some space to the menu lists and move the close button a little bit to the top to stick to the Figma design and have too better UI 👌

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @HFG43 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rustam-tolipov — APPROVED** [review]
Hi @HFG43, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Developer-512px-10

## Highlights
1. No Linters errors 💯
2. All requirements are complete 💯
3. Great design 💯 

## Optional suggestions
No optional suggestions :)

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**